terraform/10-video-hearing-wowza/modules/wowza/network.tf

```diff
@@ -1,3 +1,11 @@
+data "azurerm_client_config" "current" {
+}
+locals {
+  dns_zone_name        = var.environment == "prod" ? "platform.hmcts.net" : "staging.platform.hmcts.net"
+  peering_vnets        = ["hmcts-hub-sbox-int"] #var.environment != "prod" && var.environment != "stg" ? ["hmcts-hub-prod-int", "ukw-hub-prod-int"] : []
+  peering_subscription = "ea3a8c1e-af9d-4108-bc86-a7e2d267f49c"
```

Comment:
to save on repeat hardcoding could this be 
```
data "azurerm_subscription" "peering_target" {
    provider = azurerm.peering_target
}

peering_subscription = data.azurerm_subscription.peering_target.subscription_id
```

This could also be done at the root TF file then passed down in the variables?